### ISSUE TITLE
Face detection on different browser

### ISSUE THREAD

**Issue by habibhajar** [justadudewhohacks/face-api.js#73]
Hi firstly i want to thank you for this amazing API that help me a lot for my project !!
So: 
 *  I use it in Firefox and it works fine but sometimes it gives me this error : "a web page slows down your browser" and blocks the browser .
* When i use it in Chrome  it not work and return this error : 
![error](https://user-images.githubusercontent.com/42493217/44299314-38472100-a2eb-11e8-8baa-b7f609786432.PNG)
if you can help me to solve this !!

**Comment by justadudewhohacks** [comment]
Hi, 
you can not simply fetch files from your file system in the browser. That's why you get this error.

Either host your model file on localhost via an express server for example, or implement a filepicker to upload the model file.

**Comment by habibhajar** [comment]
Hi,
Why i can not ? either the same page works fine in Firefox !
and sorry for this question but "Either host your model file on localhost via an express server for example, or implement a filepicker to upload the model file." i don't get it and how i can do that i am a beginner in js ...

**Comment by justadudewhohacks** [comment]
Hmm, that's odd. Usually browsers don't allow you access files from your filesystem for security reason, that's why the fetch request in model.load is failing in chrome (not sure why this works for you in firefox).

If you look at my examples in this repo, you can see that I state, which folders can be publicly accessed (in [server.js](https://github.com/justadudewhohacks/face-api.js/blob/master/examples/server.js)). This way you can load your model from an url e.g. net.load('models/my-model'). You can do the same for your model.

With filepicker I mean, you can use an input element of type 'file', which is another way to browse and load files from your browser. There should be plenty of tutorials out there, explaining how to implement a file upload.

**Comment by habibhajar** [comment]
okey thanks a lot for your explanation and for the second : a web page slows down your browser" and blocks the browser : 
![error](https://user-images.githubusercontent.com/42493217/44300523-3a67aa80-a300-11e8-8217-cc1de71827d1.PNG)

**Comment by justadudewhohacks** [comment]
Isn't this just a warning? When exactly does this appear, when loading the model or during inference?

**Comment by habibhajar** [comment]
yes it is just a warning and it happen during loading the model  !
